tests/ORM/UuidableTest.php

```diff
@@ -26,7 +25,7 @@ public function testUuidLoading(): void
 
         $uuidableRepository = $this->entityManager->getRepository(UuidableEntity::class);
 
-        /** @var UuidableTrait $entity */
+        /** @var UuidableEntity $entity */
```

Comment:
```suggestion
        /** @var UuidableInterface $entity */
```

This would be correct, as Interface are source of truth. Trait are one of way to use them, but it should work without them.

Reply:
Of course, you can remove this commit and add an exception for phpstan. But this option is also correct and does not require exceptions.